apps/frontend/src/components/Avatar/Avatar.tsx

```diff
@@ -1,18 +1,30 @@
-import { UserCircleIcon } from '@heroicons/react/24/outline';
+import { PencilSquareIcon, UserCircleIcon } from '@heroicons/react/24/outline';
 import Image from 'next/image';
 
 import type { AvatarProps } from './types';
 
-export default function Avatar({ imageUri, size }: AvatarProps) {
-  return imageUri ? (
-    <Image
-      alt="Profile picture"
-      className="object-cover object-center rounded-full"
-      height={size}
-      src={imageUri}
-      width={size}
-    />
-  ) : (
-    <UserCircleIcon className={`h-[${size}] w-[${size}]`} />
+export default function Avatar({ imageUri, size, onEdit }: AvatarProps) {
+  return (
+    <div className="relative">
+      {onEdit ? (
+        <PencilSquareIcon
+          className="absolute top-[10%] right-[10%] bg-white p-[3%] rounded-full"
+          height={size / 4.5}
+          onClick={onEdit}
```

Comment:
`onClick` should be placed on a button for accessibility reasons, but not on a icon directly